great_expectations/cli/datasource.py

```diff
@@ -192,6 +192,12 @@ def _build_datasource_intro_string(datasource_count):
     help="By default open in browser unless you specify the --no-view flag",
     default=True,
 )
+@click.option(
+    "--assume-yes/--yes",
```

Comment:
I just had 2 questions regarding this section 

1) It's my understanding that the `/` separates 2 flags that enable or disable an option. Here it seems like both flags are doing the same thing. Did you intend to do this? 

2) my initial impression was that sending in the `--assume-yes` flag meant the answer to both questions 1) "Would you like to profile [DATA] [Y/n]?" and 2) "would you like to build datadocs? [Y/n], but it seems like the flag only says "yes" to the second question. What are your thoughts on making this flag a little more specific? (something like `build-docs`?)

Reply:
I think you are right. Took inspiration from this PR 
https://github.com/great-expectations/great_expectations/pull/1604 which does the same for build docs.
Making the flag more specific sounds like a good idea to me.

Reply:
Thinking more of it, I think it would be better if the  --assume-yes` flag would be consistent across commands and skips all the confirmation questions that can be skipped.

Reply:
I gave it some more thought, and I like your second point more too (keeping the --assume-yes flag and skipping all confirmation steps).  It's a pattern that we could keep for other CLI functions as well. 

Do you think you could implement it for the datasource profile command? 

Also, thanks for creating #1829 as well.

Reply:
I updated it to skip the other step as well based on assume_profile